zebra-instructions.md

```diff
@@ -10,17 +10,20 @@ These fixes disable mining pool operator payments and miner payments: they just
 
 1. [Build Zebra](https://github.com/ZcashFoundation/zebra#build-instructions)
    Zebra will need to be compiled with the `getblocktemplate-rpcs` feature:
+
     ```sh
     cargo build --release --features "getblocktemplate-rpcs"
     ```
+
 2. Configure `zebrad.toml`:
-    * change the `network.network` config to `Testnet`
-    * add your testnet transparent address in `mining.miner_address`, or you can use the ZF testnet address `t27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v`
-    * ensure that there is an `rpc.listen_addr` in the config to enable the RPC server
-    
+
+    - change the `network.network` config to `Testnet`
+    - add your testnet transparent address in `mining.miner_address`, or you can use the ZF testnet address `t27eWDgjFYJGVXmzrXeVjnb5J3uXDM9xH9v`
+    - ensure that there is an `rpc.listen_addr` in the config to enable the RPC server. For example: `rpc.listen_addr = '127.0.0.1:3000'`.
```

Comment:
Please use the standard testnet RPC port 18232, to match `pool_configs/zcash_testnet.json` and the example config below:
```suggestion
    - ensure that there is an `rpc.listen_addr` in the config to enable the RPC server. For example: `rpc.listen_addr = '127.0.0.1:18232'`.
```